src/silverlabnwb/nwb_file.py

```diff
@@ -184,6 +185,8 @@ def rel(file_name):
         self.read_user_config()
         header_fields = self.parse_experiment_header_ini(rel('Experiment Header.ini'))
         speed_data, expt_start_time = self.read_speed_data(rel('Speed_Data/Speed data 001.txt'))
+        if expt_start_time is None:
+            expt_start_time = pd.Timestamp.now()  # FIXME get experiment start time from header if not present in speed data
```

Comment:
That's an interesting one, because there seems to be some inconsistency in the data! For instance, `LabViewData2020/200225_19_09_16` has in the header
```
Experiment Date Time = "25/02/2020 18:22:17"
```
but the folder name implies a later start, as does the speed data, and all 3 differ. (Speed data starts:
```
25/02/2020	19:12:13.160587	4	0.00	332.316
```